test/views/UpdateYourEmployerInformationViewSpec.scala

```diff
@@ -17,7 +17,6 @@
 package views
```

Comment:
application stop

Reply:
None of the rest of the view specs have application.stop()